### PR TITLE
Reword wrap lines for Python 3 only migration. 

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

File: drafts/migration-to-python-3-only.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ to a Python 3 only code base, but the timeline of such a migration.
...
 # Too long, Didn't read (TL;DR:).
 
-  - High Priority: Fix Pip to refuse to install Python 3 only packages on Python 2. Extremely
-    high priority as we want pip version to be release and propagate after
-    that.
-  - High Priority: Fix Setuptools to be able to express `requires-python` in `setup.py`.
+  - High Priority: Fix Pip to refuse to install Python 3 only packages on
+    Python 2. Or to accept the `py3` only suffix on source distributions. High
+    priority as we want pip version to be release and propagate after that. But
+    it is not a requirement that have to be done before IPython 6.0.
```
Comment (fperez) [inline drafts/migration-to-python-3-only.md]: `s/have/has/`

File: drafts/migration-to-python-3-only.md [hunk 2]
```diff
@@ -191,7 +191,9 @@ have a dependency on `python3`.
 
 
 - `nbconvert` CLI: Can likely be migrated to Python 3 only in roughly same schedule than notebook.
-- `nbformat` : suggest to keep Python 2 support as long as we support IPython 5.x, maybe later, to allow people to "upgrade" their notebook and read newer version on "old" notebook.
+- `nbformat` : keep Python 2 support as long as we support IPython
+  5.x, maybe later, to allow people to "upgrade" their notebook and read newer
+  version on "old" notebook.
```
Comment (fperez) [inline drafts/migration-to-python-3-only.md]: `s/version/versions/`

**Comment by fperez** [comment]
Trivial fixes, otherwise ok

**Comment by takluyver** [comment]
You gave me push access to your fork before, IIRC, so I'll do an English pass.

File: drafts/migration-to-python-3-only.md [hunk 3]
```diff
@@ -190,8 +190,11 @@ have a dependency on `python3`.
 # Not in this roadmap:
 
 
-- `nbconvert` CLI: Can likely be migrated to Python 3 only in roughly same schedule than notebook.
-- `nbformat` : suggest to keep Python 2 support as long as we support IPython 5.x, maybe later, to allow people to "upgrade" their notebook and read newer version on "old" notebook.
+- `nbconvert` CLI: Can likely be migrated to Python 3 on roughly the same
+  schedule as the notebook.
+- `nbformat` : keep Python 2 support as long as we support IPython
+  5.x, maybe later, to help people work with newer notebook file format versions
+  and upgrade older notebooks to newer formats.
```
Comment (takluyver) [inline drafts/migration-to-python-3-only.md:197]: @Carreau is this wording what you meant? I wasn't quite sure.

Comment (Carreau) [inline drafts/migration-to-python-3-only.md:197]: Sure, I just rewrapped it, but the new wording is fine.

**Comment by Carreau** [comment]
Thanks for the changes merging.

**Pull request merged**